core-services/release-controller/_releases/release-origin-4.5.json

```diff
@@ -11,6 +11,16 @@
   "verify":{
     "aws":{
       "prowJob":{"name":"release-openshift-origin-installer-e2e-aws-4.5"}
+    },
+    "upgrade":{
+      "upgrade":true,
+      "optional":true,
+      "prowJob":{"name":"release-openshift-origin-installer-e2e-aws-upgrade"}
+    },
+    "upgrade-minor":{
+      "upgrade":true,
+      "upgradeFrom": "PreviousMinor",
+      "prowJob":{"name":"release-openshift-origin-installer-e2e-aws-upgrade"}
```

Comment:
You need to define okd jobs since the content is different.  Fork release-openshift-origin-installer-e2e-aws-upgrade to be `release-openshift-okd-installer-e2e-aws-upgrade` (like the other `okd` jobs).